### DISCUSSION TITLE
Should we say "Orders" or "Sales" ?

### DISCUSSION THREAD

**Discussion by Blair2004** [Blair2004/NexoPOS#340] [General]
We're continuously trying to make NexoPOS better for everyone. We've realized some of you rather like to name what is currently called "orders" on NexoPOS "sales". What do you think about that?

![image](https://user-images.githubusercontent.com/5265663/136706025-bdad8d8c-c734-48c3-8291-3c5931141fab.png)

**Comment by dchiputura** [comment]
I think calling them sales is better. Usually an order under normal circumstances refers to potential sale that the customer places in view of buying it later and when the order is paid then it becomes a sale.

**Reply by Blair2004** [reply]
You raised a good point.
